languages/elixir/exercises/concept/pattern-matching/.meta/design.md

```diff
@@ -0,0 +1,45 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to introduce the basics of pattern matching, focusing on matching within tuples and maps.
+
+## Learning objectives
+
+- Know what pattern matching is.
+- Know how to deconstruct a more complex data type into something simpler.
+
+## Out of scope
+
+- Head | Tail syntax in lists
+- Structs
+- Guard clauses
+- Matching on binaries
+- The pin operator
+
+## Prerequisites
+
+This exercise's prerequisites are:
+
+- `basics`: needs to be able to define named functions, return values
```

Comment:
```suggestion
- `maps`
- `tuples`
```

Reply:
maybe string-interpolations might be useful to have preceeding this, a true two-fer type exercise

Reply:
One thing to note is that by having these concepts (maps and tuples) as prerequisites, means that you can't be using pattern matching in those exercises.